packages/dds/tree/src/test/feature-libraries/editableTree.spec.ts

```diff
@@ -0,0 +1,511 @@
+    return field;
+}
+
+function buildTestPerson(): PersonType {
+    const proxy = buildTestProxy(person);
+    return proxy as PersonType;
+}
+
+function expectTreeEquals(inputField: UnwrappedEditableField, expected: JsonableTree): void {
+    assert(inputField !== undefined);
+    const expectedType = schemaMap.get(expected.type) ?? fail("missing type");
+    const primary = getPrimaryField(expectedType);
+    if (primary !== undefined) {
+        assert(Array.isArray(inputField));
+        // Handle inlined primary fields
+        const expectedNodes = expected.fields?.[primary.key];
+        if (expectedNodes === undefined) {
+            assert.equal(inputField.length, 0);
+            return;
+        }
+        expectTreeSequence(inputField, expectedNodes);
+        return;
+    }
+    // Above assert fails to narrow type to exclude readonly arrays, so cast manually here:
+    const node = inputField as EditableTreeOrPrimitive;
+    if (isPrimitiveValue(node)) {
+        // UnwrappedEditableTree loses type information (and any children),
+        // so this is really all we can compare:
+        assert.equal(node, expected.value);
+        return;
+    }
+    // Confirm we have an EditableTree object.
+    assert(node[proxySymbol] !== undefined);
+    assert.equal(node[valueSymbol], expected.value);
+    const type = node[typeSymbol];
+    assert.equal(type, expectedType);
+    for (const key of Object.keys(node)) {
+        const subNode: UnwrappedEditableField = node[key];
+        assert(subNode !== undefined, key);
+        const fields = expected.fields ?? {};
+        assert.equal(key in fields, true);
+        const field: JsonableTree[] = fields[key];
+        const isSequence = getFieldKind(getFieldSchema(type, key)).multiplicity === Multiplicity.Sequence;
+        // implicit sequence
+        if (isSequence) {
+            expectTreeSequence(subNode, field);
+        } else {
+            assert.equal(field.length, 1);
+            expectTreeEquals(subNode, field[0]);
+        }
+    }
+}
+
+function expectTreeSequence(field: UnwrappedEditableField, expected: JsonableTree[]): void {
+    assert(Array.isArray(field));
+    assert(Array.isArray(expected));
+    assert.equal(field.length, expected.length);
+    for (let index = 0; index < field.length; index++) {
+        expectTreeEquals(field[index], expected[index]);
+    }
+}
+
+describe("editable-tree", () => {
+    it("proxified forest", () => {
+        const proxy = buildTestPerson();
+        assert.ok(proxy);
+        assert.equal(Object.keys(proxy).length, 5);
+        assert.strictEqual(proxy[typeSymbol], personSchema);
+        assert.strictEqual(proxy.address[typeSymbol], addressSchema);
+        assert.strictEqual((proxy.address.phones[2] as ComplexPhoneType)[typeSymbol], complexPhoneSchema);
+        assert.strictEqual(proxy[getTypeNameSymbol]("name"), stringSchema.name);
+        assert.strictEqual(proxy.address[getTypeNameSymbol]("phones"), phonesSchema.name);
```

Comment:
The `import { strict as assert } from "assert";` at the top means `assert.strictEqual` is equivalent to `assert.equal`.
```suggestion
        assert.strictEqual(proxy[typeSymbol], personSchema);
        assert.equal(proxy.address[typeSymbol], addressSchema);
        assert.equal((proxy.address.phones[2] as ComplexPhoneType)[typeSymbol], complexPhoneSchema);
        assert.equal(proxy[getTypeNameSymbol]("name"), stringSchema.name);
        assert.equal(proxy.address[getTypeNameSymbol]("phones"), phonesSchema.name);
```

Reply:
oops, thats my mistake. Would be nice if those methods were marked as deprecated as there is no reason to call them directly.